simulation/__main__.py

```diff
@@ -34,6 +34,7 @@ def convert_to_timedelta(minutes_string):
     parser.add_argument("--history-filename", default="history.jsonl")
     parser.add_argument("--household-distribution", type=pd.read_csv)
     parser.add_argument("--heat-pump-awareness", type=float, default=0.4)
+    parser.add_argument("--annual-renovation-rate", type=float, default=0.05)
```

Comment:
Think we might need a `help=...` here with a bit more description about the rate. Might not be in this PR, but can do another one adding help statements to each of the arguments as I think it would be useful.

e.g the `rate` here refers to "per year" (from reading the code below)